base/cvd/cuttlefish/common/libs/utils/files.cpp

```diff
@@ -287,17 +287,17 @@ bool CanAccess(const std::string& path, const int mode) {
 bool IsDirectoryEmpty(const std::string& path) {
   std::unique_ptr<DIR, int (*)(DIR*)> direc(opendir(path.c_str()), closedir);
   if (!direc) {
-    LOG(ERROR) << "IsDirectoryEmpty test failed with " << path
-               << " as it failed to be open" << std::endl;
+    LOG(VERBOSE) << "IsDirectoryEmpty test failed with " << path
+                 << " as it failed to be open" << std::endl;
     return false;
```

Comment:
nit: nit: This should fail instead of returning that the directory isn't empty. It should return `Result<bool>`